srcpkgs/Komikku/template

```diff
@@ -1,19 +1,19 @@
 # Template file for 'Komikku'
 pkgname=Komikku
-version=0.29.2
-revision=2
+version=0.30.0
+revision=1
 wrksrc=Komikku-v${version}
 build_style=meson
 hostmakedepends="gettext glib-devel gobject-introspection pkg-config"
 makedepends="gtk+3-devel libhandy1-devel"
 depends="gtk+3 libhandy1 libnotify libsecret python3-BeautifulSoup4
  python3-Pillow python3-Unidecode python3-cloudscraper python3-dateparser
  python3-gobject python3-keyring python3-lxml python3-magic
- python3-pure-protobuf"
+ python3-natsort python3-pure-protobuf"
```

Comment:
python3-requests dependency is missing

Reply:
It's pulled in via `python3-cloudscraper`